sdk/python/setup.py

```diff
@@ -33,7 +33,7 @@
 
 setuptools.setup(
     name="kubeflow-training",
```

Comment:
Did you resolve https://peps.python.org/pep-0625/?

Reply:
So, your point is wondering if we should rename to "kubeflow_training" or "kubeflowtraining"?

Reply:
I think, it is fine to keep it for 1.9 SDK since in the Deprecation they say:
```
 Any source distributions already uploaded will remain in place as-is and do not need to be updated.
 ```
For the newer Kubeflow Trainer SDK, we will name it `kubeflow`.
Does it look good @tenzen-y @terrytangyuan ?

Reply:
SGTM

Reply:
Sure